db_docker/restore_db.sh

```diff
@@ -5,6 +5,10 @@ set -x
 
 if [ "$RESTORE_DB" == "TRUE" ]; then
     echo "******DEV Env - PostgreSQL initialisation******"
+    if [ ! -f $DB_DUMP_PATH ]
+    then
+        python3 /download_dump.py
```

Comment:
Did you check that downloading the dump is working this way? I had a hard time downloading the dump at restore time due to permissions to save the file if I remember correctly, that's why I made sure it happens in the Dockerfile instead.

Reply:
I cheked downloading both at build and spin up time and I worked as expected.